package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "login.dfe.validation",
-  "version": "2.0.3",
+  "version": "2.0.8",
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/jest"
+    "test": "jest --runInBand --coverage"
```

Comment:
is there any particular reason for updating the test script?

Reply:
yes if you run node version 18 and the code moves to version node 18 then the command isn't reconiszed by vsc code for windows changing it to this command will allow you to complete unit test with the npm test command